airflow_dags/open_elections/README.md

```diff
@@ -79,7 +79,7 @@ Potential vote count columns for precinct data are:
 ca_county_votes should have PK ['election_id', 'precinct', 'party', 'candidate']
```

Comment:
github did a very poor job of  displaying this changed.  Only thing I did to this file was remove a space.

out put -> output

Reply:
"Display the rich diff"